src/app/stake/page.tsx

```diff
@@ -0,0 +1,136 @@
+"use client";
+
+import { Table } from "@radix-ui/themes";
+import Image from "next/image";
+import { useRouter } from "next/navigation";
+import React from "react";
+
+import { StakeAsset } from "@/app/types/stakeAsset";
+
+const assets: StakeAsset[] = [
+  {
+    assetName: "Native Bitcoin",
+    assetSymbol: "BTC",
+    amount: 0.0001,
+    price: 10000,
+    totalBalance: 0.0001,
+    remainingBalance: 0.0001,
+  },
+  {
+    assetName: "Native ETH",
+    assetSymbol: "ETH",
+    amount: 0.0001,
+    price: 10000,
+    totalBalance: 0.0001,
+    remainingBalance: 0.0001,
+  },
+  {
+    assetName: "Staked Coinbase ETH",
+    assetSymbol: "cbETH",
+    amount: 0.0001,
+    price: 10000,
+    totalBalance: 0.0001,
+    remainingBalance: 0.0001,
+  },
+];
+
+const StakePage: React.FC = () => {
+  const router = useRouter();
+  const handleOnClick = (assetSymbol: string) => () => {
+    router.push(`/stake/${assetSymbol.toLocaleLowerCase()}`);
+  };
+
+  return (
+    <div className="flex pl-[64px] pr-[56px] h-full px-6 py-0 flex-col items-start gap-4 shrink-0 self-stretch">
```

Comment:
where possible, please use the tailwinds sizing scale. we should be leveraging their theme config to set base styling, sizing, colors, etc.

Reply:
removed these paddings